pa_config/rca.conf

```diff
@@ -49,6 +49,10 @@
       "promotion-rate-mb-per-second" : 500,
       "young-gen-gc-time-ms-per-second" : 400
     },
+    //queue rejection rca
```

Comment:
The comment is the key name itself. It may be removed.

Reply:
removed